js/features/mpc-stripes.js

```
@@ -0,0 +1,757 @@
+async function getOriginalsDir(create) {
+  return state.dirHandle.getDirectoryHandle(ORIGINALS_DIR, { create });
+}
+
+// Clean source bytes for an entry: the backup in originals/ when present,
+// otherwise the file itself (which is only clean before its first processing).
+async function readCleanFile(entry) {
+  try {
+    const dir = await getOriginalsDir(false);
+    const backup = await dir.getFileHandle(entry.name);
+    return await backup.getFile();
+  } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)